spec/rubocop/cop/rspec/file_path_spec.rb

```diff
@@ -71,119 +71,119 @@
     RUBY
   end
 
-  it 'ignores shared examples' do
+  it 'does not register an offense for shared examples' do
     expect_no_offenses(<<-RUBY, 'spec/models/user.rb')
       shared_examples_for 'foo' do; end
     RUBY
   end
 
-  it 'skips specs that do not describe a class / method' do
+  it 'does not register an offense for describe a class / method' do
```

Comment:
It is more like "does not register an offence for example groups do not describe a class / method".